### PR TITLE
Add burst count licenses & total cost for used licenses fields to O365 Bill model

### PR DESCRIPTION
**Pull request by Jchazari**
https://us-east-1.api.ilandcloud.com/doc/current/#/Companies/getO365BillingHistory

### CODE REVIEW

File: src/sdk/model/company/o365-bill.ts [hunk 1]
```diff
@@ -10,6 +10,13 @@ export class O365Bill {
   constructor(private _json: O365BillJson) {
   }
 
+  /**
+   * Get the number of licenses exceeding the reserved licenses count
+   */
```
Comment (iamtash) [inline src/sdk/model/company/o365-bill.ts:15]: `@returns` doc

File: src/sdk/model/company/o365-bill.ts [hunk 2]
```diff
@@ -52,6 +59,13 @@ export class O365Bill {
     return this._json.total;
   }
 
+  /**
+   * Get the used total including burst usage if any
+   */
```
Comment (iamtash) [inline src/sdk/model/company/o365-bill.ts:64]: `@returns` doc

**Review by iamtash — CHANGES_REQUESTED** [review]
'Added' --> 'Add' in the commit message. the general convention is to keep the action/verb (e.g., add, upgrade, fix, remove, refactor) in a commit message  in the present tense.

**Comment by Juchazari** [comment]
@iamtash I should have asked before pushing updates, but there are multiple methods without return doc, should I add them for those methods as well? Also, like you said it should be `@returns` correct? Because there are two methods I see with one having `@return` and the other having `@returns`.

**Comment by iamtash** [comment]
> @iamtash I should have asked before pushing updates, but there are multiple methods without return doc, should I add them for those methods as well? Also, like you said it should be `@returns` correct? Because there are two methods I see with one having `@return` and the other having `@returns`.

pretty sure `@return` and `@returns` are synonyms. both are fine for me, i guess as long as within a particular file the tag usage is consistent. yeah if you don't mind filling in the missing docs that'd be great 🙂

**Review by iamtash — APPROVED** [review]

**Review by ariquinones — APPROVED** [review]

**Pull request merged**